chapter_attention-mechanisms/attention-cues.md

```diff
@@ -169,6 +169,17 @@ npx.set_np()
 from d2l import torch as d2l
 import torch
 ```
+```{.python .input}
+#@tab tensorflow
+from d2l import tensorflow as d2l
+import tensorflow as tf
+```
+
+```{.python .input}
+#@tab tensorflow
+from d2l import tensorflow as d2l
+import tensorflow as tf
+```
```

Comment:
You have duplicate code here. Please remove this.

Reply:
okay